version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "0.15.4"
+version in ThisBuild := "0.15.4.1"
```

Comment:
this is changed automatically during a release

Reply:
Fixed